<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormWrong</class>
 <widget class="QWidget" name="FormWrong">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QStackedWidget{
	background-color: #343A40;
	border: 1px #343A40;
}</string>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="Picture">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>-10</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="HomeButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: #FFFFFF;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="CalendarButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: #FFFFFF;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="FriendsButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: #FFFFFF;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Inicio">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>15</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	background-color: #343A40;
	border: 0px;
	padding: 5px;
	color: #D00000;
}</string>
     </property>
     <property name="text">
      <string>Inicio</string>
     </property>
    </widget>
    <widget class="QLabel" name="Nombre">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>150</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	background-color: #343A40;
	border: 0px;
	color: #FFFFFF;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Codigo">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>30</y>
       <width>150</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	background-color: #343A40;
	border: 0px;
	color: #D00000;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>621</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_Filtrado">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>70</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>69</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btt_IgualA">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>69</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btt_Recien">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Comentar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btt_Relevant">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="CBB_Criterios">
   <property name="geometry">
    <rect>
     <x>363</x>
     <y>70</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Ver">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>93</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ver</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_Comentar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>352</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
